### DISCUSSION TITLE
Ошибка 502 - Что-то пошло не так

### DISCUSSION THREAD

**Discussion by Bondian** [Chimildic/goofy#193] [Помощь с ошибкой] [answered]
Больше десяти таких ошибок за выполнение. Боялся сожрёт 6-минутный лимит, но в итоге выполнение завершилось.
В описании ошибок ничего не нашёл...
![Скриншот 25-06-2022 223355](https://user-images.githubusercontent.com/85184404/175784679-392dfcd5-84f6-40d5-8b54-153ec18947a9.png)

**Comment by Bondian** [comment]
Должен добавить, что ошибка уже исчезла...

**Reply by Chimildic** [reply]
На их стороне проблемы. Очень неинформативный текст ошибки.

**Reply by Bondian** [reply]
Мне тоже так показалось, но подумал, что возможно о ней уже что-то известно.
Собственно, вопрос исчерпан.

**Reply by efremych** [reply]
У меня кстати текст ошибки другой
![0244A1EB-CC36-404D-87E7-1734962709C8](https://user-images.githubusercontent.com/76212904/175817699-f8567f92-3457-461c-91b7-e3f1c4224efe.jpeg)

**Reply by Chimildic** [reply]
Вообще, ошибки >= 500 ловятся на повторную отправку - `tryFetchOnce`. Судя по логу эта ветка сработала: пауза 2 секунды и отсутствие второй ошибки. Может ноль треков с этим не связан?

**Reply by efremych** [reply]
> Может ноль треков с этим не связан?

врядли. Скорее всего, уже без залогированной ошибки ноль треков каким-то образом получил запрос. Как в случае с любимыми треками. «ДО» из описания плейлиста считывается, «ПОСЛЕ» - массив треков для перезаписи. 
Всё в один день кстати вроде произошло, получается.

**Reply by Chimildic** [reply]
Почти ничего не понял из описанного.
Из того что вижу, ошибка с индексом правда проблема. А 502 же ловится для повторного запроса.

**Comment by Chimildic** [comment, marked as answer]
Возможную причину и решение описал [здесь](https://github.com/Chimildic/goofy/discussions/175#discussioncomment-3033687).
